frontend/src/App.js

```diff
@@ -1,21 +1,15 @@
-import './App.css';
+import "./App.css";
```

Comment:
actually this frontend folder can be completely ignored -  I only added it for testing the backend, should be removed

Reply:
Yes, I did ignore it, I just noticed that I accidentally deleted a line :)